logstash-input-beats.gemspec

```diff
@@ -23,7 +23,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "logstash-core-plugin-api", ">= 1.60", "<= 2.99"
 
   s.add_runtime_dependency "logstash-codec-plain"
-  s.add_runtime_dependency "concurrent-ruby", [ ">= 0.9.2", "<= 1.0.0" ]
+  s.add_runtime_dependency "concurrent-ruby", "~> 1.0"
```

Comment:
are we ok with not accepting 0.9.x? I'm guessing most logstash releases now use 0.9.x due to this restriction, will it stop this new version of being installed?